readme.md

```diff
@@ -3,13 +3,9 @@
 
 > Import WebAssembly code with Rollup
 
-Use this [Rollup plugin](https://rollupjs.org) to import source code which compiles to WebAssembly (such as C, C++, Wat, Rust), or just import standalone `.wasm` binaries.  There are a few configs to quick-start languages:
+Use this [Rollup](https://github.com/rollup/rollup) plugin to import WebAssembly modules.
 
- - `wasm.emscripten` for C and C++ using [Emscripten](https://github.com/kripken/emscripten)
- - `wasm.wabt` for WAT using [WABT](https://github.com/webassembly/wabt)
- - Others? [Submit an issue](https://github.com/jamen/rollup-plugin-wasm/issues/new)
-
-You can also create your own configs. See [Configs](#configurations) and [`src/index.js`](https://github.com/jamen/rollup-plugin-wasm/blob/master/src/index.js) for more details.
+By default, WebAssembly modules are imported as a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise), But you can use the `sync` flag to specify otherwise (See ["Sync Modules"](#sync_modules)).
```

Comment:
It's probably worth noting that Chrome disallows synchonous compilation of WebAssembly modules that are > 4KBytes